sdk/src/types/api/core.rs

```diff
@@ -325,7 +325,7 @@ pub struct CongestionResponse {
     pub reference_mana_cost: u64,
     /// The Block Issuance Credits of the requested account.
     #[serde(with = "string")]
-    pub block_issuance_credits: u64,
+    pub block_issuance_credits: i128,
```

Comment:
Not too sure about this but it can actually be negative...

Reply:
👀

Reply:
We still need the full amplitude of an u64 but it can also go negative so I guess i128 will do